File: 백준/Bronze/31495. 그게 무슨 코드니．．/그게 무슨 코드니．．.cs
string code = Console.ReadLine();

if (code.Length >= 2 && code[0] == '"' && code[^1] == '"')
{
    string result = code.Substring(1, code.Length - 2);
    if (result.Length > 0)
    {
        Console.WriteLine(result);
    }
    else
    {
        Console.WriteLine("CE");
    }
}
else
{
    Console.WriteLine("CE");
}

File: 백준/Bronze/31495. 그게 무슨 코드니．．/그게 무슨 코드니．．.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String code = sc.nextLine();

        if (code.length() >= 2 && code.charAt(0) == '"' && code.charAt(code.length() - 1) == '"') {
            String result = code.substring(1, code.length() - 1);
            if (!result.isEmpty()) {
                System.out.println(result);
            } else {
                System.out.println("CE");
            }
        } else {
            System.out.println("CE");
        }
    }
}
